ament_tools/helper.py

```diff
@@ -222,6 +222,10 @@ def extract_argument_group(args, delimiting_option):
     return trimmed_args, extracted_args
 
 
+def compute_deploy_destination(context, filename, dst_subfolder=''):
+    return os.path.join(context.install_space, dst_subfolder, filename)
```

Comment:
It's used from two places. I did this incase the method of building this changed, then you'd be forced to change it in both locations.